cheetah/accelerator.py

```diff
@@ -21,6 +21,10 @@
 class Element:
     """
     Base class for elements of particle accelerators.
+    Every subclass depending on this base class inherits the Parameters, Attributes and Functions from this base class.
+    New Parameters, Attributes and Functions can be added in the subclass and the Functions can be redefined. 
+    To sum up the base class is used to set the default settings for every subclass, as well as commenting on the behavior of 
+    various base functions, which are just slightly changed for each subclass.
```

Comment:
I think this comment is not needed as that is how classes work in Python and other programming languages, i.e. not specific to Cheetah.